packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -94,17 +94,21 @@ export class CdkToolkit {
         throw new Error(`There is no file at ${options.templatePath}`);
       }
       const template = deserializeStructure(await fs.readFile(options.templatePath, { encoding: 'UTF-8' }));
-      diffs = printStackDiff(template, stacks.firstStack, strict, contextLines, stream);
+      diffs = options.securityCheckOnly
+        ? +printSecurityDiff(template, stacks.firstStack, RequireApproval.Broadening)
+        : printStackDiff(template, stacks.firstStack, strict, contextLines, stream);
     } else {
       // Compare N stacks against deployed templates
       for (const stack of stacks.stackArtifacts) {
         stream.write(format('Stack %s\n', colors.bold(stack.displayName)));
         const currentTemplate = await this.props.cloudFormation.readCurrentTemplate(stack);
-        diffs += printStackDiff(currentTemplate, stack, strict, contextLines, stream);
+        diffs += options.securityCheckOnly
+          ? +printSecurityDiff(currentTemplate, stack, RequireApproval.Broadening)
+          : printStackDiff(currentTemplate, stack, strict, contextLines, stream);
       }
     }
 
-    return diffs && options.fail ? 1 : 0;
+    return diffs && (options.fail || options.securityCheckOnly) ? 1 : 0;
```

Comment:
Honestly -- good call, but if there's already an option for this, let's just reuse. We can always pass `cdk diff --security-check-only --fail`.

Reply:
O I was thinking that it would be something a user could use as well?

And the --fail would add more friction?